src/test/java/org/opensearch/security/dlic/dlsfls/DlsTermsLookupAsserts.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright OpenSearch Contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package org.opensearch.security.dlic.dlsfls;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.opensearch.search.SearchHit;
+
+/*
+ * Copyright OpenSearch Contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+public class DlsTermsLookupAsserts {
+
+    /**
+     * Asserts that the source map of a search hit contains an field that contains access codes
+     * and asserts those access codes contain at least on of the access codes for the user
+     * @param sourceMap
+     * @param accessCodesKey
+     * @param userCodes
+     */
+    public static void assertAccessCodesMatch(Map<String, Object> sourceMap, String accessCodesKey, Integer[] userCodes) {
+        Object field = sourceMap.get(accessCodesKey);
+        Assert.assertTrue(sourceMap.toString(), field instanceof Collection<?>);
+        Collection<?> documentAccessCodes = (Collection<?>) field;
+        // make sure the access codes in the document contain at least one code of the user access codes
+        Assert.assertTrue(sourceMap.toString(), documentAccessCodes.removeAll(Arrays.asList(userCodes)));
```

Comment:
I do not quite understand this. Why removing the elements of `userCodes` from `documentAccessCodes` ensures that the access codes in the document contain at least one code of the user access codes?